lib/govuk_component/warning_text_helpers.rb

```diff
@@ -2,15 +2,25 @@
 
 module GovukComponent
   module WarningTextHelpers
-    def govuk_warning_text(body = nil, assistive_text = t('common.warning'), **tag_options)
+    def govuk_warning_text(body = nil, assistive_text = t('common.warning'), **tag_options, &block)
```

Comment:
needs tests

Reply:
There is a test for this but I may not have checked it in when you reviewed this.